content/en/02-installing-pixie/04-install-schemes/02-yaml.md

```diff
@@ -45,20 +48,22 @@ Run the following CLI command to extract Pixie's manifest files:
   Please refer to <a href="/reference/admin/environment-configs">Environment-Specific Configurations</a> for other configurations that should be set for your specific Kubernetes environment.
 </Alert>
 
-```bash
+<TemplatedCodeBlock
+  code={`
 # Extract YAML (No OLM present on cluster).
-px deploy --extract_yaml ./ --deploy_key <PIXIE_DEPLOYMENT_KEY>
-
+export PX_CLOUD_ADDR=@PLACEHOLDER@
+px deploy --extract_yaml ./ --deploy_key <PIXIE_DEPLOYMENT_KEY> \n
 # Extract YAML (OLM already exists on cluster).
-px deploy --extract_yaml ./ --deploy_key <PIXIE_DEPLOYMENT_KEY> --deploy_olm=false
-
+export PX_CLOUD_ADDR=@PLACEHOLDER@
+px deploy --extract_yaml ./ --deploy_key <PIXIE_DEPLOYMENT_KEY> --deploy_olm=false \n
 # Extract YAML (Self-hosting Pixie Cloud).
-px deploy --extract_yaml ./ --deploy_key <PIXIE_DEPLOYMENT_KEY> --dev_cloud_namespace plc
-
+export PX_CLOUD_ADDR=@PLACEHOLDER@
```

Comment:
This option is for self-hosted cloud, so we shouldn't set the cloud addr to a hosted cloud here.

Reply:
Thanks for catching this! Updated.